terminus-ui/popover/src/popover.component.scss

```diff
@@ -0,0 +1,77 @@
+@import './../../scss/helpers/color';
+@import './../../scss/helpers/shadows';
+@import './../../scss/helpers/spacing';
+
+$PLACEMENT_MARGIN: 5px;
+
+.popover__container {
+  @include elevation-element(snackbar);
+  background-color: color(pure);
+  display: none;
+  padding: spacing(default);
+  position: absolute;
+
+  .popover__arrow {
+    border-color: color(pure);
+    border-style: solid;
+    height: 0;
+    position: absolute;
+    width: 0;
+  }
+
+  &[x-placement^='top'] {
+    margin-bottom: $PLACEMENT_MARGIN;
+    .popover__arrow {
+      border-bottom-color: transparent;
+      border-left-color: transparent;
+      border-right-color: transparent;
+      border-width: 5px 5px 0;
+      bottom: -5px;
+      left: calc(50% - 5px);
+      margin-bottom: 0;
+      margin-top: 0;
+    }
+  }
+  &[x-placement^='bottom'] {
+    margin-top: $PLACEMENT_MARGIN;
+    .popover__arrow {
+      border-left-color: transparent;
+      border-right-color: transparent;
+      border-top-color: transparent;
+      border-width: 0 5px 5px;
+      left: calc(50% - 5px);
+      margin-bottom: 0;
+      margin-top: 0;
+      top: -5px;
+    }
+  }
+  &[x-placement^='right'] {
+    margin-left: $PLACEMENT_MARGIN;
+    .popover__arrow {
+      border-bottom-color: transparent;
+      border-left-color: transparent;
+      border-top-color: transparent;
+      border-width: 5px 5px 5px 0;
+      left: -5px;
+      margin-left: 0;
+      margin-right: 0;
+      top: calc(50% - 5px);
+    }
+  }
+  &[x-placement^='left'] {
+    display: block;
+    margin-right: $PLACEMENT_MARGIN;
+    .popover__arrow {
+      border-bottom-color: transparent;
+      border-right-color: transparent;
+      border-top-color: transparent;
+      border-width: 5px 0 5px 5px;
+      margin-left: 0;
+      margin-right: 0;
+      right: -5px;
+      top: calc(50% - 5px);
```

Comment:
We should probably be using our 5px variable so all these don't need to be manually updated if our sizing changes. (your editor should be yelling at you about these 'magic numbers')

Reply:
I totally expected lint or editor would be yelling at these random numbers, but neither of them complained, so I didn't update lol

Reply:
Dang... you may need to verify your IDE is using our rules then cause I _definitely_ get yelled at 😂